### PR TITLE
Add recipe for jupyter-sphinx

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
This is ready!

File: recipes/jupyter_sphinx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  imports:
+    - jupyter_sphinx
+
+about:
+  home: https://github.com/jupyter-widgets/jupyter-sphinx
+  license: BSD 3-Clause
+  license_family: BSD
+  # license_file: LICENSE
```
Comment (jakirkham) [inline recipes/jupyter_sphinx/meta.yaml]: Would be a great if we could get a PR going to fix issue ( https://github.com/jupyter-widgets/jupyter-sphinx/issues/8 ).

Comment (SylvainCorlay) [inline recipes/jupyter_sphinx/meta.yaml]: Yes, I kept it as a comment as I missed this issue before the release.

**Comment by SylvainCorlay** [comment]
Thanks for merging!

**Pull request merged**